### DISCUSSION TITLE
Cypress open not clearing session after first describe

### DISCUSSION THREAD

**Discussion by Gayathri-Kesavakannan** [cypress-io/cypress#22647] [General]
Issue 1: Runner/ test is not refreshed for second describe. Second describe is not loading application it just retains previous test

Issue2: when I click on close in test runner execution window it's not closing runner. I need to click on "stop " in other cy window

Issue1
I have test with multiple describe where I will launch my application .
```
Describe(first)
Before{
visitandlogin(user1)
}
{
It(1){}
It(2){}
}
Describe(second)

Before{
visitandlogin(user2)
}
{
It(1){}
It(2){}
}
```
Headless it's working fine , but when I test headed I. cypress9.0 describe1 is tested but describe 2 is not loading page it's still in page of describe 1

**Comment by lmiller1990** [comment]
Hi - can you post the actual code (the JavaScript) including what `visiandlogin` does? It looks like you've just included some pseudo code - having something we can use to create a minimal reproduction would be very helpful. Thanks!
